packages/platform-sdk-eth-server/src/index.ts

```diff
@@ -3,6 +3,8 @@ import Joi from "joi";
 
 import { useClient, useDatabase, useLogger } from "./helpers";
 
+const PAGE_SIZE = 25; // TODO Do we support customizing this by the user? Like a param pageSize or limit?
```

Comment:
Do we parameterize the page size via a `limit` query param? Perhaps checking it's not greater than some default?

Reply:
We can leave this at a fixed size because those servers are designed to be only consumed by the SDK itself.